PlayerController

DreamFlightStudios opened the pull request.

DreamFlightStudios committed:
обновили контроллер

добавил глобал интсалир и еще что то

DreamFlightStudios committed:
убрал строчку лишнюю

DreamFlightStudios committed:
переименовал поле дирактион и настроил прыжок

DreamFlightStudios commented:
@Hitsar можешь смотреть

Hitsar reviewed (commented):
Ревью

Hitsar commented on Assets/Scripts/PlayerController.cs:
```diff
@@ -0,0 +1,77 @@
+        Cursor.visible = false;
+    }
+
+    private void Update()
```
Тебя не смущает что у тебя движение и IsGrounded на физике находяться в Update? InputSystem не обязательно в апдейте использовать кстати

DreamFlightStudios replied:
предлагаешь в фиксед апдейт переместить? я понял А с IsGrounded что сделать? игрок не должен двигаться когда не на земле, еще перед тем как метод движения вызывать, хочу проверку дирактиона на vextor3.null добавить, хз можно так или нет для того что бы лишний раз движение не вызывать

Hitsar replied:
В чем проблема просто весь метод в фиксед апдейт перенести? Движение камеры если рваное будет то в апдейт

DreamFlightStudios replied:
ну так я так и написал

DreamFlightStudios committed:
обновил контроллер

переименовал PlayerActions в Actions
перенес вызов метода движения в фикседапдейт, перенес инпуты в методы, где они импользуются

DreamFlightStudios committed:
фиксед на просто дельта тайм поменял
